### PR TITLE
AUX outputs param setup fix

### PR DESCRIPTION
**Pull request by LorenzMeier**

### CODE REVIEW

**Comment by LorenzMeier** [comment]
Flash overflow. So we're just at the limit.

**Comment by LorenzMeier** [comment]
@tops4u It would be great if you could test this.

**Comment by tops4u** [comment]
@LorenzMeier Checked and failed : See #5989

**Comment by LorenzMeier** [comment]
@tops4u This works now, I just tested it.

**Comment by LorenzMeier** [comment]
@julianoes Can you briefly review?

**Comment by LorenzMeier** [comment]
@kd0aij Can you review and maybe test this one?

File: src/systemcmds/pwm/pwm.c [hunk 1]
```diff
@@ -122,7 +122,7 @@ get_parameter_value(const char *option, const char *paramDescription)
 	if (strncmp("p:", option, 2) == 0) {
 
 		char paramName[32];
-		strncpy(paramName, option + 2, 16);
+		strncpy(paramName, option + 2, 17);
```
Comment (kd0aij) [inline src/systemcmds/pwm/pwm.c]: Is 17 the maximum length of a parameter name? 
Everything else looks good to me.

Comment (LorenzMeier) [inline src/systemcmds/pwm/pwm.c]: Param name + 1 (null termination). See: http://pubs.opengroup.org/onlinepubs/7908799/xsh/strncpy.html.

**Comment by LorenzMeier** [comment]
@kd0aij Any chance you could flash & take this for a short flight?

File: src/systemcmds/pwm/pwm.c (hunk 1 above)
Comment (kd0aij) [inline src/systemcmds/pwm/pwm.c]: I knew that exactly 17 bytes should be written to paramName, but I couldn't find any guarantee that option points to a string with length < 17 
Interesting that PWM_AUX_DISARMED is 16 bytes long.

**Comment by kd0aij** [comment]
Test flight went OK.

For some reason I had to recalibrate the ESCs before this would fly again, and I ran out of battery after just a minute in the air, but performance in stabilized and altctl with EKF2 was nominal. 
I had changed the fmu-v2 config to compile the new logger, but managed to lose the sys_logger parameter change, so no logger was running. Will do a longer flight tomorrow with the new logger.

**Comment by LorenzMeier** [comment]
Thanks for the baseline testing. Rebased and applied.

**Pull request closed without merging**